enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -0,0 +1,493 @@
+authors:
+  - "@bentito"
+  - "@jharrington22"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@joelanford" # for OLM, OperatorHub changes
+  - "@bparees" # for OLM, OperatorHub changes
+  - "@csrwng" # for HyperShift changes
+  - "@spadgett" # for Console changes
+approvers:
+  - "@sdodson"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@abutcher" # for CCO API changes, or please suggest alternative
+creation-date: 2023-01-21
+last-updated: 2023-02-15
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/OCPBU-4
+  - https://issues.redhat.com/browse/PORTENABLE-471
+  - https://issues.redhat.com/browse/CCO-286
+see-also:
+  - []
+replaces:
+  - []
+superseded-by:
+  - []
+---
+
+# Tokenized Authentication for Red Hat Operators on Cloud Providers
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Amazon Web Services (AWS) Secure Token Service (STS) allows you to request temporary limited-privilege credentials. A concrete 
+example is write permission on an S3 bucket on AWS. STS is supported on all the major cloud providers where OpenShift is
+deployed. Currently, several Red Hat operators provide detailed instructions for how to use the operator on STS-enabled
+clusters. The process is designed to have manual operations performed by the customer. This enhancement seeks to bring common standards to the process of enabling an
+operator to access cloud provider resources, by enabling automation, UX and logic for the process. 
+
+## Motivation
+
+Several important Red Hat operators have normal operating modes where they assure their operands have required cloud 
+resources to perform properly. Currently, when a cluster is installed on a cloud platform an admin is required to 
+follow instructions (for example [1]) to provide STS-based credentials to the operator such that needed resources can be
+accessed by the operand. This enhancement seeks to normalize this process across operators, so users of several of them
+have the same experience and similar steps to perform. We aim to automate and reduce those steps as much as possible to 
+make adding operators (via OLM) simpler on an STS enabled cluster.
+
+### User Stories
+
+* As a cluster admin I know which OLM Operators support tokenized authentication for my cloud.
```

Comment:
Is this a goal or a user story, user stories are normally formed as "As a <persona>, I want to <do something>, so that I can <achieve some goal>", I think this has the persona and goal, but not the what to do, is the intention to have some identifier on the operators or something?